pkgs/development/tools/icestorm/default.nix

```diff
@@ -1,19 +1,19 @@
-{ stdenv, fetchFromGitHub, python3, libftdi }:
+{ libcxxStdenv, fetchFromGitHub, python3, libftdi }:
 
-stdenv.mkDerivation rec {
+libcxxStdenv.mkDerivation rec {
```

Comment:
Yes, a Makefile patch is OK. Ah, I see you've made upstream issue. If upstream takes CC variable from the environment, you don't have to do any patching at all (just pass makeFlags = [ "CC=gcc" ] or something).